leftwm/doc/leftwm.1

```diff
@@ -0,0 +1,260 @@
+.I config.toml
+For a list of available commands use the '-l|list' flag.
+.IP "state"
+Prints the current state of leftwm (in JSON format). You can also use flags and liqud-like syntax for a more refined output of this command.
+.IP "theme"
+Manage leftwm themes (This is part of an external package found in: https://GitHub.com/leftwm/leftwm-theme).
+.B "NOTE:"
+This subcommand will check for basic / common mistakes made in the directory structure or permissions, however it will not check or report if any mistakes were made in any extra scripts, bar configurations, liquid templates and anything that isn't the up or down executables.
+.SH CONFIGURING LEFTWM
+.IP "Configuration file"
+The settings file of leftwm can be found at:
+.I $HOME/.config/leftwm/config.toml
+.IP
+With this file you can configure the modkey for leftwm, available keybinds, tag names, workspace layouts and more.
+.RS
+NOTE: This file is automatically generated when leftwm or leftwm-check is run for the first time.
+.SS Default keybinds
+.IP "Movement"
+ up, down, left or right refer to the arrow keys in your keyboard.
+.TS
+tab(;);
+l l.
+Keybinding;Description
+_
+Mod + (1-9);Switch to a desktop/tag
+Mod + Shift + (1-9);Move the focused window to desktop/tag
+Mod + W;Switch the desktops for each screen. Desktops [1][2] changes to [2][1]
+Mod + Shift + W;Move window to the other desktop
+Mod + (up or down);Focus on the different windows in the current workspace
+Mod + Shift + (up or down);Move the different windows in the current workspace
+Mod + Enter;Move selected window to the top of the stack in the current workspace
+Mod + Ctrl + (up or down);Switch between different layouts
+Mod + Shift + (left or right);Switch between different workspaces
+Mod + Shift + Enter;Open a terminal
+Mod + Ctrl + L;Lock the screen
+Mod + Shift + X;Exit LeftWM
+Mod + Shift + Q;Close the current window
+Mod + Shift + R;Reload LeftWM and its config
+Mod + p;Use dmenu to start application
+.TE
+.PP
+.B Note:
+Although we encourage you to use Alacritty, LeftWM will set your default terminal to the first terminal it finds in this list (in the order presented):
+.IP \[bu] 2
+Alacritty
+.IP \[bu] 2
+Termite
+.IP \[bu] 2
+Kitty
+.IP \[bu] 2
+URXVT
+.IP \[bu] 2
+RXVT
+.IP \[bu] 2
+ST
+.IP \[bu] 2
+ROXTerm
+.IP \[bu] 2
+Eterm
+.IP \[bu] 2
+XTerm
+.IP \[bu] 2
+Terminator
+.IP \[bu] 2
+Terminology
+.IP \[bu] 2
+Gnome Terminal
+.IP \[bu] 2
+XFCE4 Terminal
+.IP \[bu] 2
+Konsole
+.IP \[bu] 2
+UXTerm
+.IP \[bu] 2
+Guake
+
+.IP "About Keybind Modifiers"
+Note: All entries require a modifier, even if blank: modifier = []
+see your
+.I
+config.toml
+file for more information.
+
+.SS Floating Windows
+.PP
+You can optionally switch between tiling or floating mode for any
+window.
+.PP
+.TS
+tab(;);
+l l.
+
+Keybinding;Description
+_
+
+Mod + MouseDrag ; Switch a tiled window to floating mode
+Mod + RightMouseDrag ; Resize a window
+Drag window onto a tile ; Switch a floating window to tiling mode
```

Comment:
Shouldn't this say something like "Drag window to workspace edge" with its current behavior?